src/main/scala/fundamentals/level03/TryExercises.scala

```diff
@@ -97,23 +96,22 @@ object TryExercises {
   trait Employee
 
   /**
-    * 1. Now remove `import TryTestTypes._` from `TryExercisesTest.scala`
-    * 2. Comment out the contents of `TryTestTypes`
+    * Now remove `import TryTestTypes._` from `TryExercisesTest.scala`
     */
 
   /**
     * Create a CSV parser to safely create an Employee
     *
-    * Start by filling out the return type
+    * Start by filling out the return value
```

Comment:
Can you just remove this line? It made sense at one point because I left out the return type of the function deliberately for the students to fill in. But I have decided to be explicit.

Reply:
Removed